apps/src/lib/levelbuilder/rubrics/rubricHelper.js

```diff
@@ -120,6 +120,12 @@ function resetPositionsOfLearningGoals(keyConceptList) {
   return keyConceptList;
 }
 
+function clearNotification(setSaveNotificationText) {
+  setTimeout(() => {
+    setSaveNotificationText('');
+  }, 8500);
+}
```

Comment:
This clears the notification after 8 seconds right?

Reply:
Yes!